charts/metallb/values.yaml

```diff
@@ -42,13 +42,16 @@ prometheus:
   # certificate to be used.
   controllerMetricsTLSSecret: ""
 
-  # the service account used by prometheus
-  # required when .Values.prometheus.podMonitor.enabled == true
-  serviceAccount: ""
+  # Create rbac resources for prometheus
+  rbacPrometheus:
+    enabled: false
+    # the service account used by prometheus
+    # required when .Values.prometheus.rbacPrometheus.enabled == true
+    serviceAccount: ""
```

Comment:
moving this inside is a breaking change for users who might already have values

Reply:
Agree, I made these changes for it. I think it keep backward compatibility.